app/builders/spotlight/bootstrap_breadcrumbs_builder.rb

```diff
@@ -1,28 +1,35 @@
-# The BootstrapBreadcrumbsBuilder is a Bootstrap compatible breadcrumb builder.
-# It provides basic functionalities to render a breadcrumb navigation according to Bootstrap's conventions.
-#
-# BootstrapBreadcrumbsBuilder accepts a limited set of options:
-#
-# You can use it with the :builder option on render_breadcrumbs:
-#     <%= render_breadcrumbs :builder => Spotlight::BootstrapBreadcrumbsBuilder %>
-#
-class Spotlight::BootstrapBreadcrumbsBuilder < BreadcrumbsOnRails::Breadcrumbs::Builder
-  include ActionView::Helpers::OutputSafetyHelper
-  def render
-    if @elements.blank?
-      return ""
+module Spotlight
+  # The BootstrapBreadcrumbsBuilder is a Bootstrap compatible breadcrumb builder.
+  # It provides basic functionalities to render a breadcrumb navigation according to Bootstrap's conventions.
+  #
+  # BootstrapBreadcrumbsBuilder accepts a limited set of options:
+  #
+  # You can use it with the :builder option on render_breadcrumbs:
+  #     <%= render_breadcrumbs :builder => Spotlight::BootstrapBreadcrumbsBuilder %>
+  #
+  class BootstrapBreadcrumbsBuilder < BreadcrumbsOnRails::Breadcrumbs::Builder
+    include ActionView::Helpers::OutputSafetyHelper
+
+    def render
+      return '' if @elements.blank?
+
+      @context.content_tag(:ul, class: 'breadcrumb') do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.